internal/graph/schema/service_maintenance.graphqls

```diff
@@ -0,0 +1,41 @@
+type RunMaintenancePayload {
+	error: String
+}
+
+input RunMaintenanceInput {
+	project: String!
+	serviceName: String!
+	teamSlug: Slug!
+	environmentName: String!
+}
+
+extend type Mutation {
+	RunMaintenance(input: RunMaintenanceInput!): RunMaintenancePayload
+}
+
+extend type ValkeyInstance {
+	"Fetch maintenances for the Valkey instance."
+	maintenance: ServiceMaintenance!
+	project: String!
+}
+
+type ServiceMaintenance {
+	updates: [ServiceMaintenanceUpdate]!
```

Comment:
Er denne garantert å være under 50? Selv for tjenester som vi ikke har støtte for enda?

> The rule of thumb is that if it is possible for the list to grow to a size that is more than 50, it should be paginated.